src/for-mobx-react-query/SpreadoSetupImpl.test.tsx

```diff
@@ -0,0 +1,89 @@
+import {waitFor} from '@testing-library/react';
+import {act, renderHook} from '@testing-library/react-hooks';
+import {uniqueId} from 'lodash';
+import React from 'react';
+import {QueryClient, QueryClientProvider, useQuery} from 'react-query';
+import {SpreadoMobXStore} from '../mobx';
+import {SpreadoSetupForMobXReactQuery} from './SpreadoSetupImpl';
+
+const builtinStore: SpreadoMobXStore = new SpreadoMobXStore();
+const queryClient: QueryClient = new QueryClient();
+
+describe('SpreadoSetupForMobXReactQuery', () => {
```

Comment:
Would you refer to tests under `src/for-mobx-swr` and `src/for-redux-react-query` to help make tests in this file more lightweight?